pup-0002.md

```diff
@@ -0,0 +1,97 @@
+---
+title: Docstring Style Conventions
+author: Austin Macdonald
+created: 28-Mar-2017
+status: proposed
+---
+
+## Summary
+
+This PUP proposes the use of Google style docstrings for Pulp 3+.
```

Comment:
s/proposes the use of Google style docstrings for Pulp 3+/adopts the Google docstring style for Pulp 3+ docstrings/

I propose ^ because when merged it will be the present tense instead of a proposal.